eclass/vdr-plugin-2.eclass

```diff
@@ -15,7 +15,6 @@
 # Eclass for easing maintenance of vdr plugin ebuilds
 
 # @ECLASS-VARIABLE: VDRPLUGIN
```

Comment:
May want to add @OUTPUT_VARIABLE if ebuilds should read it

Reply:
Hi, ebuilds cant read it if they want... but not necessarily must do...

Reply:
I think that still fits the criteria. I should've said "can" or "are allowed to"

Reply:
NP, you are citating the dev manual... "This tag indicates that the variable's value (which will be set by the eclass) should be read within an ebuild " 
IMHO the dev manual should be changed. Something like "variable is set by the eclass and read-only for the ebuild" ?

Reply:
will push in a minute